appium/lib/screen-object/accessors/element.rb

```diff
@@ -81,26 +96,116 @@ def dynamic_text_exists? dynamic_text
         end
       end
 
-      def scroll
-        $driver.execute_script 'mobile: scrollTo',:element => element.ref
-        # $driver.execute_script("mobile: scroll",:direction => direction.downcase, :element => element.ref)
+      def gesture(args)
```

Comment:
Remove this `gesture` method and use the on in `ScreenObject::Accessors` to keep it dry.